modules/arm_plugin/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -62,5 +62,6 @@ std::vector<std::string> disabledTestPatterns() {
         // Sporadic hanges on linux-debian_9_arm runner (armv7l) 72140
         ".*canStartSeveralAsyncInsideCompletionCallbackWithSafeDtor.*AUTO.*",
 #endif
+        "TransformationTestsF.ConvertLayoutArmMaxPoolV8.*",
```

Comment:
Tests can be enable after this https://github.com/openvinotoolkit/openvino/pull/14813 is merged.